crates/bevy_ecs/src/component.rs

```diff
@@ -410,34 +493,43 @@ impl Components {
         index
     }
 
+    /// Gets the total number of components registered.
```

Comment:
❓ What is a registered component? Is it the same thing as an initialized component?